tests/components/google_assistant/test_trait.py

```diff
@@ -1065,15 +1065,18 @@
         'openPercent': 100
     }
 
+    # No state
+    trt = trait.OpenCloseTrait(hass, State('cover.bla', STATE_UNKNOWN, {
+    }), BASIC_CONFIG)
+
+    assert trt.sync_attributes() == {}
+
     # Assumed state
     trt = trait.OpenCloseTrait(hass, State('cover.bla', cover.STATE_OPEN, {
         ATTR_ASSUMED_STATE: True,
     }), BASIC_CONFIG)
 
     assert trt.sync_attributes() == {}
-    assert trt.query_attributes() == {
```

Comment:
Can you update the test to make sure that we raise for unknown and assumed state?

```
with pytest.raises(SmartHomeError):
    trt.query_attributes()
```